libc/include/sysexits.yaml

```
@@ -0,0 +1,9 @@
+header: sysexits.h
+header_template: sysexits.h.def
+standards:
+  - POSIX
```

sysexits.h is not in POSIX. (https://pubs.opengroup.org/onlinepubs/9799919799/ click Headers in bottom left).

I think you can just put `BSDExtensions` and `GNUExtensions`.

```suggestion
  - BSDExtensions
  - GNUExtensions
```